emf/src/test/kotlin/com/strumenta/kolasu/emf/KolasuMetamodelTest.kt

```diff
@@ -40,7 +40,6 @@ class KolasuMetamodelTest {
             "java.io", "https://strumenta.com/javaio", "javaio",
             resource
         )
-        javaIOMetamodelBuilder.provideClass(java.io.Serializable::class)
         javaIOMetamodelBuilder.generate()
```

Comment:
Does it still make sense to include a package for java.io that doesn't contain any classes in the StarLasu metamodel?

Reply:
You are right, we should remove it